doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,55 @@
 
 <h3>New features since last release</h3>
 
+* The `qml.fourier.reconstruct` function is added. It can be used to
+  reconstruct one-dimensional Fourier series with a minimal number of calls
+  to the original function.
```

Comment:
```suggestion
  reconstruct QNodes outputting expectation values along a specified
  parameter dimension, with a minimal number of calls to the
  original QNode. The returned
  reconstruction is exact and purely classical, and can be evaluated
  without any quantum executions.
```

Reply:
I know technically the reconstruction works for _arbitrary_ functions, as long as they can be represented as Fourier series, but I think the above might help users identify where this functionality would be useful in the context of quantum computing?